patches/ui-base-accelerators-accelerator.cc.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/ui/base/accelerators/accelerator.cc b/ui/base/accelerators/accelerator.cc
+index c33829d85745e..6aa6e376f44db 100644
+--- a/ui/base/accelerators/accelerator.cc
++++ b/ui/base/accelerators/accelerator.cc
+@@ -388,11 +388,11 @@ std::u16string Accelerator::ApplyLongFormModifiers(
+   // Note that we use 'else-if' in order to avoid using Ctrl+Alt as a shortcut.
+   // See https://devblogs.microsoft.com/oldnewthing/20040329-00/?p=40003 for
+   // more information.
++  BRAVE_UI_BASE_ACCELERATOR_APPLY_LONG_FORM_MODIFIERS_
+   if (IsCtrlDown())
+     result = ApplyModifierToAcceleratorString(result, IDS_APP_CTRL_KEY);
+   else if (IsAltDown())
```

Comment:
At first, I thought removing `else` would be better instead of adding more condition.
but it makes `Alt+Ctrl+H`. Starting with `Ctrl+` seems more conventional way.

Reply:
Yeah, I did the same thing :cry: